src/desktop/apps/artwork/client/footer.jade

```diff
@@ -1,2 +1,3 @@
 include ../components/related_artworks/index
 include ../components/related_artists/index
+- foo = 'bar'
```

Comment:
lol what this

Reply:
Ah yes -- sometimes when templates are only executed client-side deep nested includes don't trigger recompiles on on change, but the root one will which then cascades. Def a bug somewhere in the Pug loader.